external/storm-jdbc/README.md

```diff
@@ -29,13 +57,21 @@ The default is set to value of topology.message.timeout.secs and a value of -1 w
 You should set the query timeout value to be <= topology.message.timeout.secs.
 
  ```java
-Config config = new Config();
-config.put("jdbc.conf", hikariConfigMap);
-JdbcInsertBolt userPersistanceBolt = new JdbcInsertBolt("jdbc.conf",simpleJdbcMapper)
+Map hikariConfigMap = Maps.newHashMap();
```

Comment:
minor thing: 
How about changing this line to 
```
Map<String, String> hikariConfigMap = new HashMap<String, String>();
```
to remove dependency about Guava from example?